apps/bookmark/save.html

```diff
@@ -47,6 +47,7 @@ <h1 id="edit-bookmark-header" data-l10n-id="edit-bookmark-title"></h1>
       </header>
       <form id="bookmark-form">
         <img id="bookmark-icon">
+        <a id="bookmark-url"></a>
```

Comment:
Assuming that a person already comes from this URL, an <input> element might make more sense than a link.
But if this is called from an app that wants you to create a bookmark, this may not be entirely true.

Maybe we should allow editing by default and provide a visit button on the side? I hope @benfrancis has an opinion about this.